impeller/entity/entity_pass.cc

```diff
@@ -589,9 +589,7 @@ EntityPass::EntityResult EntityPass::GetEntityForElement(
 
     auto offscreen_texture_contents =
         subpass->delegate_->CreateContentsForSubpassTarget(
-            subpass_texture,
-            Matrix::MakeTranslation(Vector3{-global_pass_position}) *
-                subpass->xformation_);
+            subpass_texture, subpass->xformation_.Basis());
```

Comment:
Is this a different bug or a new bug?

Reply:
Investigating this a bit to remind myself about how we got here. It's kind of peculiar that this was working before.

Reply:
I reverted this little change -- it turns out the test was actually behaving correctly and just needed to be adjusted. Before this PR, the effect_transform wasn't being set by the ImageFilter proc defined in the test.